### ISSUE TITLE
environment variables showing up as nil

### ISSUE THREAD

**Issue by deniseschannon** [rancher/rancher#5806]
**Rancher Version:**
v1.2.0-pre2
rancher-compose v0.9.2

**Steps to Reproduce:**

1. set an environment variable `export MYSQL_ROOT_PASSWORD=test`
2. start a service using rancher-compose

```
dbtest:
  image: mysql
  environment:
    MYSQL_ROOT_PASSWORD: 
```

**Results:** 
In the API, the environment variable is `<nil>`.

**Expected:**
The environment variable should be `test`.

**Comment by cmeury** [comment]
Seeing this with 1.1.3 as well.

**Comment by sra** [comment]
I'm seeing this as well with rancher-compose > 0.9.0

**Comment by deniseschannon** [comment]
Verified with v0.9.3-rc1 and Rancher v1.2.0-pre3-rc1

**Comment by andyshinn** [comment]
Will there be a release of 0.9.3 that is compatible with Rancher 1.1.4 or is this fix only in Rancher Compose 0.10+?
